lldb/source/Interpreter/CommandObject.cpp

```diff
@@ -715,7 +715,7 @@ Thread *CommandObject::GetDefaultThread() {
     return nullptr;
 }
 
-bool CommandObjectParsed::Execute(const char *args_string,
+void CommandObjectParsed::Execute(const char *args_string,
                                   CommandReturnObject &result) {
   bool handled = false;
```

Comment:
Not necessary for this patch, but you could probably get rid of `handled` with some an return.